sdcm/sct_provision/aws/layout.py

```diff
@@ -28,17 +28,22 @@ def _test_config(self):
         return TestConfig()
 
     def provision(self):
+        use_scylla_cloud = (self._params.get('cluster_backend') == 'xcloud' or
+                            self._params.get('xcloud_provisioning_mode'))
+
         if self.placement_group:
             self.placement_group.provision()
         SCTCapacityReservation.reserve(self._params)
         SCTDedicatedHosts.reserve(self._params)
-        if self.db_cluster:
+
+        # skip DB cluster provisioning for Scylla Cloud
+        if not use_scylla_cloud and self.db_cluster:
             self.db_cluster.provision()
         if self.monitoring_cluster:
             self.monitoring_cluster.provision()
         if self.loader_cluster:
             self.loader_cluster.provision()
-        if self.cs_db_cluster:
+        if not use_scylla_cloud and self.cs_db_cluster:
```

Comment:
I would suggest to let it fail, if someone tries to configure oracle on cloud, we don't support it in the test itself anyhow (just for aws backend).